action/lint.go

```diff
@@ -0,0 +1,160 @@
+package action
+
+import (
+	"github.com/google/go-github/github"
+	"github.com/helm/helm/chart"
+	"github.com/helm/helm/log"
+	"github.com/helm/helm/manifest"
+	"gopkg.in/yaml.v2"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
```

Comment:
The recommendation is to split core libraries and "other" libraries.

namespace imports (`. "foo"`) and side effect imports (`_ "foo"`) often are set apart.

I've also seen internal libraries moved to the bottom, but that's not ubiquitous.